infra/modules/service/variables.tf

```diff
@@ -55,7 +55,6 @@ variable "db_vars" {
   description = "Variables for integrating the app service with a database"
   type = object({
     security_group_ids         = list(string)
-    access_policy_arn          = string
```

Comment:
good catch. do we need to delete [this line](https://github.com/navapbc/platform-test/blob/main/infra/app/service/main.tf#L102) too

<img width="764" alt="image" src="https://github.com/navapbc/platform-test/assets/447859/c401fc94-9d53-4ca8-968a-d9ef5487ba1e">

Reply:
Yes, that should be deleted. I must not have committed that change

Reply:
Doesn't show up in [the file](https://github.com/navapbc/platform-test/blob/9f743d068626d164368c8b7170d859829c782258/infra/app/service/main.tf#L96)